src/main/java/frc/robot/subsystems/arm/ArmState.java

```diff
@@ -4,39 +4,134 @@
+	INTAKE(Rotation2d.fromDegrees(186), ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND, ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED),
+	ALGAE_OUTTAKE(
+		Rotation2d.fromDegrees(185),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	PRE_L1(
+		Rotation2d.fromDegrees(190),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	L1(
+		Rotation2d.fromDegrees(190),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	PRE_L2(
+		Rotation2d.fromDegrees(-13),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	L2(
+		Rotation2d.fromDegrees(-13),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	PRE_L3(
+		Rotation2d.fromDegrees(10),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	L3(
+		Rotation2d.fromDegrees(10),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	PRE_L4(
+		Rotation2d.fromDegrees(55),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	L4(
+		Rotation2d.fromDegrees(-25),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	LOW_ALGAE_REMOVE(
+		Rotation2d.fromDegrees(-30),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	HIGH_ALGAE_REMOVE(
+		Rotation2d.fromDegrees(-30),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	PRE_NET(
+		Rotation2d.fromDegrees(60),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	NET(
+		Rotation2d.fromDegrees(45),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	PROCESSOR_OUTTAKE(
+		Rotation2d.fromDegrees(180),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	),
+	CLIMB(
+		Rotation2d.fromDegrees(-11),
+		ArmConstants.CRUISE_VELOCITY_ANGLES_PER_SECOND,
+		ArmConstants.ACCELERATION_ANGLES_PER_SECOND_SQUARED
+	);
+	// @formatter:on
 
 	private final Rotation2d position;
+	private final Rotation2d maxVelocityRotation2dPerSecond;
+	private final Rotation2d maxAccelerationRotation2dPerSecondSquared;
 
-	ArmState(Rotation2d position) {
+	ArmState(Rotation2d position, Rotation2d maxVelocityRotation2dPerSecond, Rotation2d maxAccelerationRotation2dPerSecondSquared) {
 		this.position = position;
+		this.maxVelocityRotation2dPerSecond = maxVelocityRotation2dPerSecond;
+		this.maxAccelerationRotation2dPerSecondSquared = maxAccelerationRotation2dPerSecondSquared;
```

Comment:
Because the angular acceleration and velocity are the same for each and every state, you can get it back to what it was, and then change the constructor

```suggestion
	ArmState(Rotation2d position) {
		this.position = position;
		this.maxVelocityRotation2dPerSecond = CRUISE_VELOCITY_ANGLES_PER_SECOND;
		this.maxAccelerationRotation2dPerSecondSquared = ACCELERATION_ANGLES_PER_SECOND_SQUARED;
```

Reply:
it is not allways, on some states we will change it, look at the new constractor